### PR TITLE
vmscrapeconfig: support max_scrape_size option

### PR DESCRIPTION
**Pull request by AndrewChubatiuk**
Related to https://github.com/VictoriaMetrics/VictoriaMetrics/pull/6434

### CODE REVIEW

**Review by f41gh7 — DISMISSED** [review]
LGTM

**Review by f41gh7 — APPROVED** [review]
LGTM

File: api/v1beta1/vmstaticscrape_types.go [hunk 1]
```diff
@@ -63,6 +63,9 @@ type TargetEndpoint struct {
 	// Timeout after which the scrape is ended
 	// +optional
 	ScrapeTimeout string `json:"scrapeTimeout,omitempty"`
+	// MaxScrapeSize defines a maximum size of scraped data for a job
+	// +optional
+	MaxScrapeSize string `json:"max_scrape_size,omitempty"`
```
Comment (f41gh7) [inline api/v1beta1/vmstaticscrape_types.go]: Btw, WDYT of including it into the vm specific params `VMScrapeParams`?

Comment (AndrewChubatiuk) [reply api/v1beta1/vmstaticscrape_types.go]: updated

**Review by f41gh7 — DISMISSED** [review]
LGTM

**Comment by AndrewChubatiuk** [comment]
@f41gh7 
ssince 1.102.0 is released, suppose this PR can be merged

**Pull request merged**